<?xml version="1.0"?>
<launch>
    <arg name="serialport" default="/dev/motor_tank" />
    <arg name="baudrate" default="115200" />
    <arg name="wheel_diameter" default="0.08" />
    <arg name="base_width" default="0.4" />
    <arg name="encoder_ticks_per_rev" default="4000" />
    <arg name="linear_coef" default="800" />
    <arg name="angular_coef" default="180" />

    <node name="tank_driver" pkg="tank_driver" type="tank_driver.py">
        <param name="serialport" value="$(arg serialport)" />
        <param name="baudrate" value="$(arg baudrate)" />
        <param name="wheel_diameter" value="$(arg wheel_diameter)" />
        <param name="base_width" value="$(arg base_width)" />
        <param name="encoder_ticks_per_rev" value="$(arg encoder_ticks_per_rev)" />
        <param name="linear_coef" value="$(arg linear_coef)" />
        <param name="angular_coef" value="$(arg angular_coef)" />
    </node>
</launch>
